### DISCUSSION TITLE
How do you activate the new admin?

### DISCUSSION THREAD

**Discussion by kudapara** [solidusio/solidus#5606] [New Admin UI/UX]
How do you activate the new admin?

>  Additionally, new installations will now be able to take advantage of the new interface right away.
    - https://solidus.io/blog/solidus-v4-3

I installed the latest version (4.3.1) and it is still showing me the legacy UI even though the show legacy UI is not turned on

Compare and contrast with the [demo and the blog](https://solidus.io/blog/solidus-v4-3)

<img width="1423" alt="Screenshot 2024-01-18 at 21 43 15" src="https://github.com/solidusio/solidus/assets/13519184/a913ac65-97a6-462e-bb33-d3bd9afee248">

![image](https://github.com/solidusio/solidus/assets/13519184/9ba5f217-7078-464e-a07a-06546f7bde4d)

**Comment by spaghetticode** [comment]
I'm thinking this may be caused by the fact that `rails g solidus:install` seems to install version `0.0.0` of Solidus Admin, at least on my machine. This version of the gem is empty, so its installer is empty. You may have seen  something like this during the install process:

<img width="861" alt="Screenshot 2024-01-19 at 15 20 34" src="https://github.com/solidusio/solidus/assets/141220/3ef6d08c-0bf9-45c5-9e66-ad9ceb7b6cdd">

if that's the case, you can change `solidus_admin` version in the `Gemfile` to `~> 0.2` and `bundle` again. After that, you'll still need to run `rails g solidus_admin:install` in order to see the new admin, and make the toggle work. 

Please let me know if this fixes your issue, thank you for reporting 🙏

**Reply by kudapara** [reply]
That worked for me, thanks 🙏
